### ISSUE TITLE
SYNO.Mesh.Network.WANStatus generates error

### ISSUE THREAD

**Issue by stephan1827** [nioc/synology-srm-nodejs-api#1]
Hi,
I am using the sample and get the following error message "error (main) API does not exist" for the command const wanStatus = await client.getWanStatus()

The rest of the script executes fine. I am using a RT6600 router with SRM 1.3.1. Update 8

**Comment by nioc** [comment]
Hello, you're right, I can not find it in the GUI either, maybe this method has been deprecated.

I will update the library in order to request the `SYNO.Core.Network.Router.ConnectionStatus` which replies something more precise:

Request:
``` json
{
    "api": "SYNO.Core.Network.Router.ConnectionStatus",
    "method": "get",
    "version": 1
},
```
Reply:
``` json
{
	"ipv4": {
		"conn_status": "normal",
		"ifname": "eth0",
		"ip": "192.168.1.16",
		"pppoe": false,
		"vpn_profile": ""
	},
	"ipv6": {
		"conn_status": "not available",
		"ifname": "",
		"ip": "",
		"pppoe": true
	}
}
```

**Comment by stephan1827** [comment]
Thx for the quick response. ConnectionStatus seems to work fine. Is there any kind of documentation what API function calls are available?

**Comment by nioc** [comment]
From Synology, not to my knowledge, I look at the HTTP requests launched from the GUI...
For this library, you can look in the [test file](https://github.com/nioc/synology-srm-nodejs-api/blob/master/test/index.test.js#L224) for mocked API replies.

**Comment by nioc** [comment]
@stephan1827 Fix is available in version 1.1.1 on npm

**Comment by stephan1827** [comment]
The function getWifiDevices also throws the same error. I looked through the web code, but I could not find an equivalent function.
 
The function getMeshNodes also throws an error, I think the version in the function must be changed from 3 to 4

**Comment by nioc** [comment]
@stephan1827 Fix for both method is available in version 1.1.2 on npm.
I also update sample code in README.MD.
